journals/apps/journals/management/commands/publish_journals.py

```diff
@@ -44,6 +44,7 @@ def _handle_create(self, options):
         try:
             journal = Journal.objects.create_journal(name, org, access_length)
             journal_about_page = self._update_wagtail_pages(journal, create=True, publish=publish)
+            journal.refresh_from_db()  # since we are updating journal object in signals.
```

Comment:
this is uncessary. we create a new Journal object right above with the name passed in, no need to refresh from db

Reply:
actually in line 46: 
`journal_about_page = self._update_wagtail_pages(journal, create=True, publish=publish)`
Publish signal is triggered  (that actually try to `patch` Journal to other services) and signal sync the journal object name with the JournalAboutPage title.
(at that time JournalAboutPage title have additional 'About page' string concatenated)

and from [line 69](https://github.com/edx/journals/blob/8d6b57a7ed0d76db56181cbb91d0ef5061ea6616/journals/apps/journals/management/commands/publish_journals.py#L69) to so on we actually `post` Journal on other services (discovery, e-commerce)